spec/requests/civil_merits_questions_swagger_spec.rb

```diff
@@ -0,0 +1,92 @@
+require "swagger_helper"
+
+RSpec.describe "civil_merits_questions", type: :request do
+  let(:data) do
+    {
+      request_id: "7d47425b-800a-41ef-a917-934acc09b50d",
+      proceedings: [example_data],
+    }
+  end
+  let(:example_data) do
+    {
+      ccms_code: "DA001",
+      delegated_functions_used: true,
+      client_involvement_type: "A",
+    }
+  end
+
+  path "/civil_merits_questions" do
+    post("Request civil merits questions for a set of proceedings") do
+      tags "Merits tasks"
+      response(200, "successful") do
+        consumes "application/json"
+        produces "application/json"
+        parameter name: :data,
+                  in: :body,
+                  schema: {
+                    type: :object,
+                    required: %w[request_id proceedings],
+                    description: "An array of proceeding data",
+                    properties: {
+                      request_id: { type: :string,
+                                    example: "2a614e1a-a3d5-49ca-949a-c3248483d274",
+                                    description: "Client generated request id that will be echoed back in the response" },
+                      proceedings: {
+                        type: :array,
+                        description: "something something data",
+                        items: {
+                          type: :object,
+                          required: %i[ccms_code delegated_function_used client_involvement_type],
+                          properties: {
+                            ccms_code: { type: :string,
+                                         description: "A code uniquely identifying the proceeding_type",
+                                         example: "SE013" },
+                            delegated_functions_used: { type: :boolean,
+                                                        description: "A boolean indicating whether delegated functions were used",
+                                                        example: true },
+                            client_involvement_type: { type: :string,
+                                                       description: "A code uniquely identifying the client_involvement_type",
+                                                       example: "A" },
+                          },
+                          description: "Array of CCMS codes, df_used and client involvement type of proceeding types to  be queried",
+                        },
+                      },
+                    },
+                  }
+        response(400, "Bad request") do
+          description "Return questions base on proceeding codes, client involvement type and delegated function use"
+          let(:example_data) do
+            {
+              ccms_code: "DA001",
+              delegated_functions_used: true,
+              client_involvement_type: "X",
+            }
+          end
+
+          # TODO: @colinbruce 10 Oct 2022 review on: 11 Oct 2022
```

Comment:
do we still want this TODO?

Reply:
No! It's something I was considering but I have removed it for now :+1: